x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details.js

```diff
@@ -183,11 +184,54 @@ function getDetailsItems(anomaly, examples, filter) {
     });
   }
 
+  items.push({
+    title: (
+      <EuiToolTip
+        position="left"
+        content={i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.recordScoreTooltip', {
+          defaultMessage:
+            'A normalized score between 0-100, which indicates the relative significance of the anomaly record result.',
```

Comment:
In the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-get-record.html), we also say "Unlike initial_record_score, this value will be updated by a re-normalization process as new data is analyzed." Perhaps it's worth adding that here too, in case users don't click on both tooltips to see the difference? For example:

```suggestion
            'A normalized score between 0-100, which indicates the relative significance of the anomaly record result. This value might change as new data is analyzed.',
```

Reply:
Done in https://github.com/elastic/kibana/pull/108216/commits/fbb27179f192a4dba74ff02620f046627b8cf8cf